### DISCUSSION TITLE
G14 2024 Fan Speed Strange Behavior?

### DISCUSSION THREAD

**Discussion by indiepaleale** [seerge/g-helper#3370] [General]
I have recently noticed that on my g14 2024, without applying custom fan curve, the CPU fan just maxed out around 4800rpm even in Turbo mode.

Here is the situation, in Turbo mode If I run cinebench, the CPU goes up to 95C, and the CPU fan will start to ramp up, at the begining it will overshoot to about 5400rpm, and then hovers back down to 4800rpm and stays there. Meanwhile the CPU temps max out at 95 the whole time, the fans just don't wanna spin faster.

If I clicked appy custom fan curve (without adjusting the default fan curve), then it behaves more like it should, and goes up to 6200rpm.

This is turbo mode without custom fan curve, been running at 95C for about 5 minutes

![Screenshot 2024-11-13 135649](https://github.com/user-attachments/assets/afdcd8e1-f1cb-4808-9161-265cb4585e5b)

This is after apply custom fan curve, goes up to 6000rpm quickly

![Screenshot 2024-11-13 135827](https://github.com/user-attachments/assets/6c3a52d4-92f2-4840-8e41-3b7bae6ad175)

Here's what I tried so far and none of them seems to change this

- Downgrade and reinstall BIOS to Version 304 and back to 306
- Downgrade and reinstall Asus System Contol Interface V3
- Install Armoury Crete
- Reinstal different versions of Ghelper
- Update Windows

So I'm really confused right now, not sure if this is normal. But I didn't notice a difference in daily use or when gaming as when cpu and gpu are running, it never maxes out to 95C. Only when I want to benchmark the CPU to see if there is any thermal issue. I remembered in Cinebench R23, the CPU hit more than 170000 in turbo mode without any custom tweaking, now I need to apply custom fan curve to keep the fan running fast to achieve this.

If you also have a G14 or G16, it would be really helpful to check if it behave the same and report back here. Thanks a lot!

**Comment by indiepaleale** [comment]
BTW, It's the same behavoir if using Armoury Crete tested on my machine.

**Comment by seerge** [comment]
@indiepaleale hello, fans are always controlled by bios, even when custom fan curve is set. It simply means that Asus has programmed them that way.

From my observations they would slowly ramp up to a higher speeds even on default profiles.

**Reply by indiepaleale** [reply]
Thank you so much ! @seerge
